docker-compose.yml

```diff
@@ -83,6 +83,16 @@ services:
     environment:
       << : *default-environment
 
+  chrome:
```

Comment:
@srowlands need to add additional volumes here, so that changes to the tests could be available in the test container without rebuilding it.